dev_setup.sh

```diff
@@ -0,0 +1,7 @@
+export SF_ROOT=$(realpath $(dirname $BASH_SOURCE))
+export ICEBOX=$SF_ROOT/third_party/icestorm/icebox
+export VPR=$SF_ROOT/../vtr-verilog-to-routing/vpr/vpr
+
+export PATH=$SF_ROOT/env/conda/bin:$PATH
+export PATH=$SF_ROOT/../arachne-pnr/bin:$PATH
+#add icestorm to conda for icepack or build from 3rd party?
```

Comment:
conda probably.  File an issue on conda-packages?

Reply:
Oops I didn't mean to add this file. But yeah I think that is something I should go back and check if it's still an issue.